packages/commonwealth/server/migrations/20230530140014-undo-role-migration.js

```diff
@@ -48,14 +63,13 @@ module.exports = {
         END
         WHERE id IN (${ids.join(', ')})
         `,
-          { transaction: t }
+          {
+            transaction: t,
+            isolationLevel: Sequelize.Transaction.ISOLATION_LEVELS.SERIALIZABLE,
+          }
         );
 
-        // Add is_user_default to Addresses from RoleAssignments table
-        const isUserDefault = await queryInterface.sequelize.query(
-          `SELECT address_id, is_user_default FROM "RoleAssignments" WHERE is_user_default = true`,
-          { type: queryInterface.sequelize.QueryTypes.SELECT, transaction: t }
-        );
+        const isUserDefault = roleInfo.filter((r) => r.is_user_default);
```

Comment:
This wont work because roleInfo excludes member roles. We need to do another query.

Reply:
I think roleInfo includes member roles due to the OR query:
WHERE is_user_default = true OR name != 'member'

I could make separate queries since it might be more efficient, but it could cause another point of deadlock

Reply:
OH I missed that line. NVM you're good.